tooling/template-base/pages/dashboard/index.page.tsx

```diff
@@ -15,24 +15,38 @@ const documentProps = {
   description: 'The catalog page'
 };
 
-function Page() {
+const AppearanceContext = createContext<Appearance | undefined>(undefined);
```

Comment:
While this approach works, we have some tools handy so we don't need to create unnecessary contexts.

We already have a `usePageContext` function that contains all the data passed to the page:

https://github.com/thoughtindustries/helium/blob/staging/tooling/template-base/renderer/usePageContext.tsx

And in `helium-server`, we add that `appearance` info to the context (there's also some other handy info like the current URL, `heliumEndpoint`, `currentUser`, etc):

https://github.com/thoughtindustries/helium/blob/e1829b250205646b6b8d82653f36c74611c7edcf/tooling/helium-server/src/utilities/init-page-context.ts#L46

So in `DisplayListView` and `GridView`, we can just import that `usePageContext`, call it, and access our `appearance` info without the need for additional context. Apologies for not pointing that one out earlier!